frontend/styles/components/form-inputs.css

```diff
@@ -0,0 +1,19 @@
+@layer components {
```

Comment:
Creator of tailwind is going to hate you =))

Reply:
WHYYYYYY 😱

Reply:
I get it now HAHAHAHA! It's a premature extraction. Plus I created a CSS file where I end up thinking for a class name.